src/main/java/io/confluent/connect/elasticsearch/bulk/BulkProcessor.java

```diff
@@ -324,6 +353,34 @@ public void flush(long timeoutMs) {
     } finally {
       flushRequested = false;
     }
+    if (log.isDebugEnabled()) {
```

Comment:
okay this guard is definitely not necessary.

Reply:
Well, I thought it was because `time.seconds()` is not free, and actually has a surprisingly non-trivial overhead.

Reply:
This is only triggered during a flush, which is a very infrequent operation.
I've found that the only log messages and operations that affect performance are in the tight loop being executed per-record.

Reply:
ack